model/security/get_certificates/structures.smithy

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@input
+structure GetCertificates_Input {}
+
+@output
+structure GetCertificates_Output {
+    certificateslist: CertificatesList
```

Comment:
As can be seen [here](https://opensearch.org/docs/latest/security/access-control/api/#example-response-38), this should be:
```suggestion
    http_certificates_list: CertificatesList,
    transport_certificates_list: CertificatesList
```